docs/sources/setup-grafana/configure-security/configure-authentication/grafana/index.md

```diff
@@ -3,7 +3,8 @@ aliases:
   - ../../../auth/grafana/
 description: Grafana OAuthentication Guide
 title: Configure Grafana authentication
-weight: 1000
+menuTitle: Grafana authentication
```

Comment:
Should we remove the word 'authentication', because none of the other menutitles include that word?